src/store/tests/test-data/cat-b/vehicle-checks/__tests__/vehicle-checks.reducer.spec.ts

```diff
@@ -0,0 +1,157 @@
+import { CatBUniqueTypes } from '@dvsa/mes-test-schema/categories/B';
+import { CompetencyOutcome } from 'src/app/shared/models/competency-outcome';
+import { VehicleChecksQuestion } from 'src/app/providers/question/vehicle-checks-question.model';
```

Comment:
Path aliases can be used on above two imports

Reply:
changed